csrfguard/src/main/resources/csrfguard.properties

```diff
@@ -46,7 +46,7 @@ org.owasp.csrfguard.configuration.provider.factory = org.owasp.csrfguard.config.
 org.owasp.csrfguard.Enabled = true
 
 # If csrf guard filter should check even if there is no session for the user
-# Note: this changed around 2014/04, the default behavior used to be to 
+# Note: this changed around 2014/04, the default behavior used to be to
```

Comment:
Please revert the white space changes because it does not relate to the functionality you have implemented, hence it shouldn't be part of this PR.